snippets/editorconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "root": {
```

Comment:
Before I merge, can you (@pbnj) confirm if this "root" snippet is supposed to be the "boilerplate" that is added to the `.editorconfig` file?

Reply:
That is correct. The `root` stanza is a boilerplate for an `.editorconfig` file

Reply:
Awesome thanks!